While ssh'd into a pre that's suspended, powerd spams all users with messages
every second or so.  This patch silences those.

Index: /usr/sbin/suspend_action
===================================================================
--- .orig/usr/sbin/suspend_action
+++ /usr/sbin/suspend_action
@@ -1,9 +1,9 @@
 #!/bin/sh
 
-echo -e "powerd: System going to sleep now\n" | wall 
+# echo -e "powerd: System going to sleep now\n" | wall 
 
 # Initiate suspend
 echo mem > /sys/power/state
 
-echo -e "powerd: System woke up.\n" | wall 
+# echo -e "powerd: System woke up.\n" | wall 
 
